apps/server/src/modules/common-cartridge/common-cartridge-client/card-client/card-client.adapter.spec.ts

```diff
@@ -0,0 +1,93 @@
+import { faker } from '@faker-js/faker';
+import { createMock, DeepMocked } from '@golevelup/ts-jest';
+import { REQUEST } from '@nestjs/core';
+import { Test, TestingModule } from '@nestjs/testing';
+import { AxiosResponse } from 'axios';
+import { Request } from 'express';
+import { UnauthorizedException } from '@nestjs/common';
+import { CardListResponse, CardResponse } from './cards-api-client/models';
+import { CardClientAdapter } from './card-client.adapter';
+import { BoardCardApi } from './cards-api-client';
+
+const jwtToken = faker.string.uuid();
```

Comment:
const jwtToken = faker.string.alphanumeric(20);
and can be moved to descripe block

Reply:
Done